hazelcast/src/test/java/com/hazelcast/test/JmxLeakHelper.java

```diff
@@ -34,29 +36,55 @@ private JmxLeakHelper() {
         // No-op.
     }
 
+    public static Collection<ObjectInstance> getBeansByClass(Class clazz) {
```

Comment:
I can't see where this method is being used.

Reply:
ok just seen in PR description.

Reply:
It is used in the other pending PR: https://github.com/hazelcast/hazelcast-enterprise/pull/3465